src/main/java/com/spotify/reaper/core/RepairSchedule.java

```diff
@@ -16,6 +16,7 @@
 import com.google.common.collect.ImmutableList;
 import com.spotify.reaper.core.RepairSegment.State;
 import com.spotify.reaper.storage.postgresql.LongCollectionSQLType;
+import java.util.UUID;
 
 import java.util.List;
 import java.util.UUID;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)